interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is software thats open-source which allows you to keep track of your source code and GitHub is the cloud service Git repository. GitHub allows for code hosting and collaboration for developers.
 
-Researched answer:
+Researched answer: Git is a version control system which is open-source and helps you manage the history of your source code, while Github is the cloud hosting service which manages your repositories from Git. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The JavaScript operators that will return a Boolean value are logical operators.
 
-Researched answer:
+Researched answer: ThecJavaScript operators that will return a Boolean values are AND, OR, and NOT. These values are considered logical operators.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: The index is the amount of time to skip, counting from the start of an array. Values are basically portions seperate bits of information that play different roles.
 
-Researched answer:
+Researched answer: The index is the placement of strings and their property length that are accounted for. Strings are considered "zero indexed" therefore the counting of the characters begins with 0 and is counted from there. A value is a chunk of information when you set a variable, it can be any data type and can be changed at any time.
```

Comment:
In this case, index and value is referring to how it correlates to arrays.  An index is the location of a specific element and the value is the content at that location.